### DISCUSSION TITLE
Math in incremental bullet points

### DISCUSSION THREAD

**Discussion by mk1564** [marp-team/marp#513] [Q&A] [answered]
Hi, it seems like math equation block with $$ sign below a bullet point starting with * sign (i.e. incremental bullet point) does not appear at the same time as the words in the same bullet point. Is it possible to fix it? Thanks!

**Comment by yhatt** [comment, marked as answer]
Aren't you missing an indent in the math block to align with the list item? If the indent was aligned, the math block should appear at the same time as the paragraph above.

<table width="100%">
<thead>
<tr>
<th>❌ Wrong</th>
<th>✅ Correct</th>
</tr>
</thead>
<tbody>
<tr>
<td>

```markdown
* abc
$$x = {-b \pm \sqrt{b^2-4ac} \over 2a}$$
```

</td>
<td>

```markdown
* abc
  $$x = {-b \pm \sqrt{b^2-4ac} \over 2a}$$
```

</td>
</tr>
</tbody>
</table>

In the correct case, there are leading 2 spaces before the math block.

**Reply by yhatt** [reply]
- **CommonMark Spec**
  - https://spec.commonmark.org/0.31.2/#example-255
  - https://spec.commonmark.org/0.31.2/#example-256

**Reply by mk1564** [reply]
Thanks for your help!
